.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,65 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: ["kind/bug", "needs-triage"]
```

Comment:
I felt it would be good to have "needs-triage" label created so that we can know which bugs needs triaging. Once triaged we can remove this. We should create this label first

Reply:
sounds good to me